### ISSUE TITLE
New release/Tagged commit?

### ISSUE THREAD

**Issue by cultab** [ray-x/lsp_signature.nvim#304]
Hello! First off, I really enjoy using this plugin, but I had to configure my plugin manager to use the latest commit instead of the latest tag (stable version) because I needed a certain fix (the trim() is nil one). 

Since the last tagged commit was made over 2 years ago maybe it's time for a version bump?

If there isn't actually a "stable" commit between 2 years ago and now then feel free to disregard this!

TL;DR: By default the Lazy plugin manager installs a 2 year old version because it's the last tagged version.

**Comment by ray-x** [comment]
A new release created [v0.3.0](https://github.com/ray-x/lsp_signature.nvim/releases/tag/v0.3.0)
